### PR TITLE
[Merged by Bors] - chore(Analysis/Seminorm): make statement of `neg_mem_ball_zero` iff

### PR DESCRIPTION
**Pull request by s1m7u**
---
As per #7987 make statement of `neg_mem_ball_zero` an iff statement.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/Seminorm.lean [hunk 1]
```diff
@@ -963,11 +963,11 @@ theorem closedBall_smul_closedBall (p : Seminorm 𝕜 E) (r₁ r₂ : ℝ) :
...
 #align seminorm.closed_ball_smul_closed_ball Seminorm.closedBall_smul_closedBall
 
--- Porting note: TODO: make that an `iff`
-theorem neg_mem_ball_zero (r : ℝ) {x : E} (hx : x ∈ ball p 0 r) : -x ∈ ball p 0 r := by
-  simpa only [mem_ball_zero, map_neg_eq_map] using hx
+theorem neg_mem_ball_zero (r : ℝ) {x : E} : x ∈ ball p 0 r ↔ -x ∈ ball p 0 r := by
+  simp only [mem_ball_zero, map_neg_eq_map]
 #align seminorm.symmetric_ball_zero Seminorm.neg_mem_ball_zero
 
+
```
Comment (Parcly-Taxel) [inline Mathlib/Analysis/Seminorm.lean:970]:
```suggestion
```

**Review by Parcly-Taxel — APPROVED** [review]

File: Mathlib/Analysis/Seminorm.lean [hunk 2]
```diff
@@ -963,9 +963,8 @@ theorem closedBall_smul_closedBall (p : Seminorm 𝕜 E) (r₁ r₂ : ℝ) :
   exact (norm_nonneg _).trans ha
 #align seminorm.closed_ball_smul_closed_ball Seminorm.closedBall_smul_closedBall
 
--- Porting note: TODO: make that an `iff`
-theorem neg_mem_ball_zero (r : ℝ) {x : E} (hx : x ∈ ball p 0 r) : -x ∈ ball p 0 r := by
-  simpa only [mem_ball_zero, map_neg_eq_map] using hx
+theorem neg_mem_ball_zero_iff (r : ℝ) {x : E} : x ∈ ball p 0 r ↔ -x ∈ ball p 0 r := by
+  simp only [mem_ball_zero, map_neg_eq_map]
 #align seminorm.symmetric_ball_zero Seminorm.neg_mem_ball_zero
```
Comment (Parcly-Taxel) [inline Mathlib/Analysis/Seminorm.lean:968]:
```suggestion
#align seminorm.symmetric_ball_zero Seminorm.neg_mem_ball_zero_iff
```

**Review by YaelDillies — APPROVED** [review]
maintainer merge

**Comment by sgouezel** [comment]
bors r+
Thanks!

**Pull request closed without merging**